### PR TITLE
Update private

### PR DESCRIPTION
**Pull request by Loyalsoldier**
As per https://github.com/v2fly/v2ray-core/pull/228

### CODE REVIEW

**Review by EpLiar — APPROVED** [review]

File: data/private [hunk 1]
```diff
@@ -5,9 +5,19 @@ local
...
-# example
-# invalid
+test
+example
+invalid
+
+# References:
+# https://www.iana.org/assignments/special-use-domain-names/special-use-domain-names.xhtml
+# https://unix.stackexchange.com/questions/92441/whats-the-difference-between-local-home-and-lan
+localdomain
```
Comment (Vigilans) [inline data/private:15]: Isn't '.local' domain included?

Comment (Loyalsoldier) [reply data/private:15]: It is at the second line of the file.

File: data/private [hunk 2]
```diff
@@ -5,9 +5,19 @@ local
...
+lan
+home.arpa
+
+# Dotless domains
+regexp:^[^.]+$
 
 # OpenWrt LuCI Web Interface
 openwrt.lan
```
Comment (Robot-DaneelOlivaw) [inline data/private:23]: Since we have `lan` now, `openwrt.lan` can be removed.

Comment (Loyalsoldier) [reply data/private:23]: https://github.com/v2fly/domain-list-community/pull/230

**Pull request merged**